src/gui/widgets/text_box_base.cpp

```diff
@@ -48,7 +51,8 @@ text_box_base::text_box_base(const implementation::builder_styled_widget& builde
 	, text_changed_callback_()
 {
 	auto cfg = get_control(control_type, builder.definition);
-	text_.set_family_class(cfg->text_font_family);
+//	text_.set_family_class(cfg->text_font_family);
```

Comment:
Why is this commented out? It's obsoleted by the new line, I'm guessing? Is there some reason to leave it in? I'd generally prefer not to have commented out code, unless there's a good reason for it.

Reply:
Removed.